components/input_environmental_personal.py

```diff
@@ -262,25 +262,54 @@ def input_environmental_personal(
     model_inputs = Models[selected_model].value.inputs
     model_inputs = convert_units(model_inputs, units)
 
-    values: ModelInputsInfo
-    for values in model_inputs:
-
-        if (
-            values.id == ElementsIDs.met_input.value
-            or values.id == ElementsIDs.clo_input.value
-        ):
-            inputs.append(create_autocomplete(values))
+    all_inputs = {
+        ElementsIDs.t_db_input.value: "Air Temperature",
```

Comment:
you should definitely NOT define this attributes here in the code. Please delete this code